Sourced from commit:
https://github.com/OrangeShark/guile-commonmark/commit/3aee5628641e2bd38274aa1ef741fda7bc7fa843

(Omitted change to guix.scm, which does not exist in the dist tarball...)

diff --git a/configure.ac b/configure.ac
index ea73570..1ba5210 100644
--- a/configure.ac
+++ b/configure.ac
@@ -5,7 +5,7 @@ AC_CONFIG_AUX_DIR([build-aux])
 AC_CONFIG_MACRO_DIR([m4])
 AM_INIT_AUTOMAKE([-Wall -Werror foreign])
 
-GUILE_PKG([2.2 2.0])
+GUILE_PKG([3.0 2.2 2.0])
 GUILE_PROGS
 if test "x$GUILD" = "x"; then
    AC_MSG_ERROR(['guild' binary not found; please check your guile-2.x installation.])
diff --git a/m4/guile.m4 b/m4/guile.m4
index 89823e9..bc0daaf 100644
--- a/m4/guile.m4
+++ b/m4/guile.m4
@@ -47,8 +47,8 @@
 # for an available version of Guile.
 #
 # By default, this macro will search for the latest stable version of
-# Guile (e.g. 2.2), falling back to the previous stable version
-# (e.g. 2.0) if it is available.  If no guile-@var{VERSION}.pc file is
+# Guile (e.g. 3.0), falling back to the previous stable version
+# (e.g. 2.2) if it is available.  If no guile-@var{VERSION}.pc file is
 # found, an error is signalled.  The found version is stored in
 # @var{GUILE_EFFECTIVE_VERSION}.
 #
@@ -61,7 +61,7 @@
 #
 AC_DEFUN([GUILE_PKG],
  [PKG_PROG_PKG_CONFIG
-  _guile_versions_to_search="m4_default([$1], [2.2 2.0 1.8])"
+  _guile_versions_to_search="m4_default([$1], [3.0 2.2 2.0])"
   if test -n "$GUILE_EFFECTIVE_VERSION"; then
     _guile_tmp=""
     for v in $_guile_versions_to_search; do
@@ -221,7 +221,7 @@ AC_DEFUN([GUILE_SITE_DIR],
 # as well.
 #
 # By default, this macro will search for the latest stable version of
-# Guile (e.g. 2.2). x.y or x.y.z versions can be specified. If an older
+# Guile (e.g. 3.0). x.y or x.y.z versions can be specified. If an older
 # version is found, the macro will signal an error.
 #
 # The effective version of the found @code{guile} is set to
@@ -237,7 +237,7 @@ AC_DEFUN([GUILE_SITE_DIR],
 AC_DEFUN([GUILE_PROGS],
  [_guile_required_version="m4_default([$1], [$GUILE_EFFECTIVE_VERSION])"
   if test -z "$_guile_required_version"; then
-    _guile_required_version=2.2
+    _guile_required_version=3.0
   fi
 
   _guile_candidates=guile

